handler/src/test/java/io/netty/handler/ssl/SslUtilsTest.java

```diff
@@ -75,4 +75,28 @@ public void testIsTLSv13Cipher() {
         assertFalse(SslUtils.isTLSv13Cipher("TLS_DHE_RSA_WITH_AES_128_GCM_SHA256"));
     }
 
+    @Test
+    public void should_get_packet_length_of_gmssl_protocol_from_ByteBuf() {
```

Comment:
This is java ... please use camel case :) This is true for all method names.

Reply:
our java project test case use this style name,  it's easy to read for long name, :)  
ok, i'll change to camel case name.

Reply:
`should_get_packet_length_of_gmssl_protocol_from_ByteBuf`
`shouldGetPacketLengthOfGmsslProtocolFromByteBuf`  

camel case name is not easy to read, :)

Reply:
done